Update readme for bindings v0.0.8

nighthawk9 opened the pull request:
I would also appreciate suggestions on changing the organization of the new keybindings.

nighthawk9 committed:
Update readme for bindings v0.0.8

Ambrevar commented on README.org:
```diff
@@ -27,14 +27,37 @@ will setup the new key bindings for you.
+| Show log                        | gl        |             |
+| Select other view               | gv        |             |
+| Save attachement                | p         |             |
+| Save multiple attachments       | P         |             |
```
You can remove this line, from mu4e 1.0 onward the two functions are an alias.

Ambrevar commented:
What do you mean with "organization"?

I suggest we also add `yu` as `mu4e-view-save-url`.

The table in the comment section of the file needs to be updated as well.

nighthawk9 committed:
Remove alias command save-attachment

nighthawk9 committed:
Add view-save-url bind

nighthawk9 committed:
Update keybinds in docs to v0.0.8

nighthawk9 committed:
Revert "Update keybinds in docs to v0.0.8"

This reverts commit 78c7dc6b11bfe16ede88f0d8aa4d204375283c7c.

nighthawk9 committed:
Update in-file docs v0.0.8

nighthawk9 commented:
Updated the comment section of the elisp code. Added the save URL to both the readme and the elisp code.

What I mean by organization is that currently the key bindings do not stay constant from the main mode map -> header mode map -> view mode map. This leaves me with the thought of: should we wait for the rest of the keybindings to be fleshed out? Or make three separate tables for the keybindings? 

As of right now, I don't think the "noting" solution is as clean as it could be.

I am relatively new to the extending features of git (rebasing, squashing, bisecting and cherry picking) to have the cleanest commit log for a branch, as well as contributing to open source projects. I thought I would start contributing to projects I use often to learn the ropes.

I appreciate the help.

Ambrevar commented:
Well, it's 3 different modes with different bindings, with a bunch in common.
The more consistent the better.  I'm quite happy with the current bindings, but if you think there is anything we could improve, don't hesitate to send a pull request.

As for git, a good history is a history made of atomic commits: one change one commit.
In this case, it's simple enough that all commits can be squashed into just one.
You can use the "rebase interactively" feature of git and then "squash" or "fixup" all commits but the first.

I also recommend Magit if you are not using it already.

Ambrevar commented:
@nighthawk9: Let me know if you want to rebase or if I should squash your commits and merge.

nighthawk9 commented:
Feel free to squash the commits and merge.

I appreciate your help. Thank you.
